factory/factory.go

```diff
@@ -43,7 +43,7 @@ func InitConfigFactory(f string) error {
 	return nil
 }
 
-func UpdateAmfConfig(f string) error {
+func UpdateConfig(f string) error {
```

Comment:
Though I agree with this change there are plenty of variables with name amfConfig... But I do not want those many variable changes.

Reply:
We can make those changes in another PR such that a single does not become to big to review. What do you think?

Reply:
Changing function name is enough. Sometimes its easy to keep amf in the variable name. I would not recommend that change.